_posts/2021-12-23-Fortran-lang-2021-in-review.md

```diff
@@ -0,0 +1,366 @@
+
+## Conferences and papers
+
+### FortranCon 2021
+
+Like in 2020, the Fortran event of this year was
+[FortranCon 2021](https://tcevents.chem.uzh.ch/event/14/), the international
+Fortran conference, held on September 23-24.
+The keynote speaker this year was [Damian Rouson](https://github.com/rouson),
+the head of the
+[Computer Languages and Systems Software (CLaSS)](https://crd.lbl.gov/divisions/amcr/computer-science-amcr/class/)
+group at the Lawrence Berkeley National Lab. 
+Fortran-lang had a dedicated session (a Fortran-lang minisymposium) on the
+second day of the conference, with the talks about specific Fortran-lang
+projects as well as the Google Summer of Code student presentations.
+FortranCon was hosted at the University of Zurich (UZH), and organized by
+[Tiziano Müller](https://github.com/dev-zero) from UZH and
+[Alfio Lazzaro](https://github.com/alazzaro) from Hewlett-Packard Enterprise.
+You can watch all FortranCon 2021 talks [here](https://www.youtube.com/playlist?list=PLeKbr7eYHjt5UaV9zQtY24oEbne9_uFni).
+
+### PackagingCon 2021
+
+Another big event for Fortran-lang was
+[PackagingCon 2021](https://packaging-con.org/), a conference dedicated to
+package management software and its community of developers.
+[Sebastian Ehlert](https://github.com/awvwgk) presented fpm--you can watch the
+recording below:
+
+<center>
+<iframe width="560" height="315" src="https://www.youtube.com/embed/YG8zEM1lAVM" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</center>
+
+### Papers
+
+This year we also wrote our first paper documenting the motivation and goals of
+Fortran-lang: "Toward Modern Fortran Tooling and a Thriving Developer
+Community" by Milan Curcic, Ondřej Čertík, Brad Richardson, Sebastian Ehlert,
+Laurence Kedward, Arjen Markus, Ivan Pribec, and Jérémie Vandenplas
+([https://arxiv.org/abs/2109.07382](https://arxiv.org/abs/2109.07382)).
+
+<center>
+<a href="https://arxiv.org/abs/2109.07382">
+<img width="600" src="{{ site.baseurl }}/assets/img/Fortran-lang-2021-in-review/fortran-lang-paper.png">
+</a>
+</center>
+
+If you use any of the Fortran-lang projects for your publications,
+please cite this paper as:
+
+```bibtex
+@article{curcic2021toward,
+  title={Toward Modern Fortran Tooling and a Thriving Developer Community},
+  author={Curcic, Milan and
+          {\v{C}}ert{\'\i}k, Ond{\v{r}}ej and
+          Richardson, Brad and
+          Ehlert, Sebastian and
+          Kedward, Laurence and
+          Markus, Arjen and
+          Pribec, Ivan and
+          Vandenplas, J{\'e}r{\'e}mie},
+  journal={arXiv preprint arXiv:2109.07382},
+  year={2021}
+}
+```
+
+We have also submitted a paper draft to IEEE's
+[Computing in Science & Engineering (CiSE)](https://ieeexplore.ieee.org/xpl/RecentIssue.jsp?punumber=5992)
+journal, titled "The State of Fortran" and led by
+[Laurence Kedward](https://github.com/lkedward).
+Stay tuned for its publication.
```

Comment:
```suggestion
For this paper, we have
[publicly invited on Discourse](https://fortran-lang.discourse.group/t/fortran-lang-community-paper/1232)
anybody in the Fortran-lang community to participate.
Once published, this will be another paper that we will recommend citing in your Fortran work.
Stay tuned for its publication.
```